### ISSUE TITLE
Tune Parameters (DNN)

### ISSUE THREAD

**Issue by liuyifly06** [liuyifly06/bubble-counting#11]
- [ ] **Tune Batch Size, Batch number and Steps first !!!!!!!**
- [ ] Batch number / Labeling speed
- [ ] Probability Multiplier / Weight function / Loss function / Optimizier
- [ ] Hidden layer numbers / Number of Nodes 
- [ ] Training echo number / Time
- [ ] Learning Rate / Decay function
- [ ] Image Patch Size
- [ ] Center Probability Mode ('PRO') versus Sum of Probability Mode ('NUM')
- [ ] Dropout
- [ ] Regularization
